yarn.lock

```diff
@@ -6955,6 +6955,17 @@ fast-glob@^3.2.9, fast-glob@^3.3.2:
     merge2 "^1.3.0"
     micromatch "^4.0.4"
 
+fast-glob@^3.3.3:
```

Comment:
someone should convince them to use tinyglobby like other packages

Reply:
I’ll talk to Lars 👍